plugins/frontend/backstage-plugin-shortcut/src/components/Widgets/StoriesCard.tsx

```diff
@@ -0,0 +1,237 @@
+    flexWrap: 'nowrap',
+  },
+  title: {
+    fontWeight: Number(theme.typography.fontWeightMedium),
+    paddingBottom: '1rem',
+  },
+  subtitle: {
+    color: theme.palette.text.secondary,
+  },
+  subtitleStarted: {
+    color: theme.palette.status.ok,
+  },
+  subtitleNotStarted: {
+    color: '#c9bb82',
+  },
+  subtitleIcon: {
+    display: 'flex',
+  },
+  linkStyle: {
+    textDecoration: 'none',
+    color: theme.palette.type === 'light' ? '#FFFFFF' : '#000000',
+  },
+  bug: {
+    backgroundColor: '#eb474717',
+    color: '#eb4747',
+    borderRadius: '6px',
+    marginLeft: '0.5rem',
+  },
+  feature: {
+    backgroundColor: '#99853317',
+    color: '#c9bb82',
+    borderRadius: '6px',
+    marginLeft: '0.5rem',
+  },
+  chore: {
+    backgroundColor: '#3d81f51f',
+    color: '#9fadc6',
+    borderRadius: '6px',
+    marginLeft: '0.5rem',
+  },
+  subtitleContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  link: {
+    color: theme.palette.text.primary,
+    '&:hover': {
+      color: '#539bf5',
+      fill: '#539bf5',
+    },
+  },
+}));
+
+export const getStoryTypeIcon = (story: string) => {
+  switch (story) {
+    case 'feature':
+      return (
+        <Tooltip title="Feature">
+          <Star fontSize="small" />
+        </Tooltip>
+      );
+    case 'bug':
+      return (
+        <Tooltip title="Bug">
+          <BugReport fontSize="small" />
+        </Tooltip>
+      );
+    case 'chore':
+      return (
+        <Tooltip title="Chore">
+          <Build fontSize="small" />
+        </Tooltip>
+      );
+    default:
+      return null;
+  }
+};
+
+const StoryItem = ({ story }: { story: Story }) => {
+  const classes = useStyles();
+  return (
+    <Box margin={1} padding={2} className={classes.listStyle}>
+      <Grid item xs={12}>
+        <Grid item>
+          <Typography variant="body1" className={classes.title}>
+            <Link
+              className={classes.link}
+              target="_blank"
+              rel="noopener noreferrer"
+              underline="none"
+              href={story.app_url}
+            >
+              {story.name}
+            </Link>
+          </Typography>
+        </Grid>
+        <Grid container className={classes.subtitleContainer} xs={12}>
+          <Grid item className={`${classes[story.story_type]}`}>
+            <Typography variant="caption" className={classes.subtitleIcon}>
+              {getStoryTypeIcon(story.story_type)} #{story.id}
+            </Typography>
+          </Grid>
+          <Grid item>
+            <Typography
+              variant="caption"
+              className={
+                story.started
+                  ? classes.subtitleStarted
+                  : classes.subtitleNotStarted
+              }
+            >
+              {story.started ? 'In progress' : 'Not started'}
+            </Typography>
+          </Grid>
+        </Grid>
+      </Grid>
+    </Box>
+  );
+};
+
+const StoriesCard = () => {
```

Comment:
Could you pls add tests for this one? It seems this does quiet some things. I think adding some basic ones to check the happy path and sad path would be nice.